.storybook/stories/FaceControls.stories.tsx

```diff
@@ -13,20 +14,26 @@ export default {
   decorators: [withKnobs, (storyFn) => <Setup cameraFov={60}>{storyFn()}</Setup>],
 }
 
-export const FaceControlsSt = ({ eyes }) => (
-  <>
-    <color attach="background" args={['#303030']} />
-    <axesHelper />
-
-    <FaceLandmarker>
-      <FaceControls eyes={eyes} />
-    </FaceLandmarker>
+function FaceControlsScene({ eyes }) {
+  return (
+    <>
+      <color attach="background" args={['#303030']} />
+      <axesHelper />
+
+      <React.Suspense fallback={null}>
```

Comment:
I guess `React.Suspense` is now required here, since `<FaceLandmarker>` provider is now suspended?